### DISCUSSION TITLE
CDN Import Errors with WaveSurfer

### DISCUSSION THREAD

**Discussion by pokemon963** [katspaugh/wavesurfer.js#2755] [Q&A] [answered]
Hi! I have been using WaveSurfer for my website. However, today I am getting errors with importing it. Specifically, I am getting this error: Failed to load resource: the server responded with a status of 520 (). Would anyone know how to fix this?

**Comment by prakharpbuf** [comment, marked as answer]
_I think_ unpkg is down.

For now, to fix this you can add the version number explicitly in your unpkg cdn link. For example,

`<script src="https://unpkg.com/wavesurfer.js/dist/plugin/wavesurfer.cursor.min.js"></script>`

becomes

`<script src="https://unpkg.com/wavesurfer.js@6.6.2/dist/plugin/wavesurfer.cursor.min.js"></script>`

Not having a version number will always pull the latest changes from the CDN. 
With 6.6.3 released just two hours ago, I don't think unpkg has been updated with it because it seems to be down. That's why when adding the version number explicitly, we provide 6.6.2 instead of the latest 6.6.3

**Reply by prakharpbuf** [reply]
I might be completely wrong tho

**Reply by prakharpbuf** [reply]
looks like its back up
